easybuild/easyconfigs/m/MDSplus/MDSplus-7.0.67-foss-2016a-Java-1.7.0_79-Python-2.7.11.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'MDSplus'
+version = '7.0.67'
+versionsuffix = '-Java-%(javaver)s-Python-%(pyver)s'
+
+homepage = 'http://mdsplus.org/'
+description = """MDSplus is a set of software tools for data acquisition and storage and a methodology for management of complex scientific data."""
```

Comment:
@SimonPinches lines should be shorter than 120 characters according to our 'coding' style, so please break it up, something like:

```python
description = """MDSplus is a set of software tools for data acquisition and storage and a methodology
 for management of complex scientific data."""
```